tests/conftest.py

```diff
@@ -45,3 +45,26 @@ def copy_case(path, config_file):
 
     with tmpdir.as_cwd():
         yield copy_case
+
+
+@pytest.fixture()
+def unused_tcp_port(unused_tcp_port, monkeypatch):
+    # The unused_tcp_port checks against localhost when providing an unused port:
+    # https://github.com/pytest-dev/pytest-asyncio/blob/9246f5825f589ff01f1c67620ce34bdc416c5af3/pytest_asyncio/plugin.py#L490
+    # In find_available_port, we use our public ip address if a current_host is
+    # not provided (as given in _get_ip_address). Using the public ip address
+    # is fine in production, but for unit testing paired with unused_tcp_port
+    # we have to specify localhost. We wrap unused_tcp_port and mock
+    # the `_get_ip_address` to only provide localhost.
+    import ert_shared.port_handler
+    import socket
+
+    def mocked_get_ip_adress():
+        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        s.connect(("0.0.0.0", 80))
+        return s.getsockname()[0]
```

Comment:
```suggestion
    def mocked_get_ip_adress():
        return "127.0.0.1"
```

Reply:
Nono, that's just making it too complicated...

Reply:
I think also `0.0.0.0` will bind to all possible interfaces, so they're not equivalent.